### PR TITLE
switch to two columns with updating text and tooltips

### PR DESCRIPTION
**Pull request by pacarole**

### CODE REVIEW

File: asset/script/main.js [hunk 1]
```diff
@@ -54,6 +56,56 @@
         }
     };
 
+    function contextOutputControl (element) {
+
+      $(document).on('lint-results', function( event, results) {
+          var inputEl = $(document).find('#post-input')[0];
```
Comment (antislice) [inline asset/script/main.js]: I have mixed feelings about the name "inputE1" unless it means something I don't know about.

**Comment by antislice** [comment]
Aside from my comment & whatever the merge conflict is, this looks good to me.

File: asset/script/main.js (hunk 1 above)
Comment (pacarole) [inline asset/script/main.js]: It's actually InputEl (as in InputElement) - but happy to write that out fully. @antislice

**Comment by pacarole** [comment]
fixed merge conflict

**Pull request merged**